configuration/globalproperties/i18n_gp.xml

```diff
@@ -0,0 +1,16 @@
+<config>
```

Comment:
Same remark here. What is different from the Haiti file?

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/4/commits/be61da316052fc30be7d24f34667c66cbd27b74c)